Common/Subworlds/Passes/FlatWorldPass.cs

```diff
@@ -16,7 +17,7 @@ public class FlatWorldPass(int floorY = 500, bool spawnWalls = false, FastNoiseL
 
 	protected override void ApplyPass(GenerationProgress progress, GameConfiguration configuration)
 	{
-		progress.Message = "ENTERING TIER X MAP"; // Sets the text displayed for this pass
+		progress.Message = Language.GetTextValue($"Mods.{PoTMod.ModName}.Generation.Terrain"); // Sets the text displayed for this pass
```

Comment:
I see this was changed but not seeing a new entry in the localization files, did this exist already? Everything else looks fine.

Reply:
Yeah this exists. Unless you specifically want the "ENTERING TIER X MAP", this is just "Generating surface".